tests/contracts/testContracts.nim

```diff
@@ -118,7 +117,13 @@ ethersuite "Marketplace contracts":
     switchAccount(client)
     let missingPeriod = periodicity.periodOf(await ethProvider.currentTime())
     await ethProvider.advanceTime(periodicity.seconds)
-    check await marketplace
-      .markProofAsMissing(slotId, missingPeriod)
-      .confirm(1)
-      .reverts("Slot not accepting proofs")
+    try:
```

Comment:
This test doesn't really cover the changes introduced, although I don't know how much we really want to cover *all* of them. We could add a test that calls `requestState` with a non-existent `requestId`, which should return `none RequestState`.